about/changelog.md

```diff
@@ -8,6 +8,43 @@ keywords: [changelog, upgrades, updates, releases]
 
 All the latest features and updates to Timescale products.
 
+<Label type="date">September 12, 2024</Label>
+
+
+## Ease of use updates for Tiered Storage
+* Do you have a really large Continuous Aggregate with lots of historical data that is no longer actively queried?
+* You can now add a tiering policy directly on a Continuous Aggregate: add_tiering_policy( <continuous_aggregate name > )
+* You can now drop columns on tiered hypertables. Previously, you could not drop columns after tiering data. The only limitation is that you cannot add a column with the same name (as the dropped column) to the hypertable.
+  
+## Live-migration tooling new releases v0.0.26 and v0.0.27
+Includes the following improvements:
+* Parallel VACUUM ANALYZE for Hypertable Chunks
+* Fixed ctid-based same table copy concurrency
+* Enhanced query compatibility with PG12
+* Resolved deadlock during pipeline sync
+* Added warning and skip setting for REPLICA IDENTITY FULL on CAGGs
+* Included rds_tools in extension skip list
+  
+## Multiple HA replicas
+Scale and Enterprise customers can now configure two new multiple High Availability (HA) Replica options directly through the console:
+* Two HA Replicas (both asynchronous) - Our highest availability configuration
+* Two HA Replicas (one asynchronous, one synchronous) - Our highest data integrity configuration
+Previously, Timescale offered only a single synchronous replica for customers seeking High Availability. The single HA option is still available.
+More details on multiple HA replicas here (https://docs.timescale.com/use-timescale/latest/ha-replicas/high-availability/).
+<include images>
+## SQL Editor now displays connection status
+<include the GIF here> I requested for this one from Justin
+Ease of use updates for Tiered Storage (Timescale Cloud exclusive)
+* Do you have a really large Continuous Aggregate with lots of historical data that is no longer actively queried?
+* You can now add a tiering policy directly on a Continuous Aggregate:
+* add_tiering_policy( <continuous_aggregate name > )
+* You can now drop columns on tiered hypertables. Previously, you could not drop columns after tiering data. The only limitation is that you cannot add a column with the same name (as the dropped column) to the hypertable.
+Other improvements
```

Comment:
Seems like this content was duplicated from [above](https://github.com/timescale/docs/pull/3431/files#diff-63c6b70086b042afa1ea56365a0350589ae4a0d4ed6618d5f46d0ab77c45fedfR14-R17)? I think it can probably be deleted.
```suggestion

```

Reply:
fixes